client/a8c-for-agencies/sections/agency-tier/primary/agency-tier-overview-v2/index.tsx

```diff
@@ -24,7 +33,12 @@ export default function AgencyTierOverviewRevamped() {
 				</LayoutHeader>
 			</LayoutTop>
 
-			<LayoutBody>Content</LayoutBody>
+			<LayoutBody>
+				<AgencyTierOverviewContent
```

Comment:
I'm not sure when the agency Tier value is null, but if that happens, I assume we have different content for that case?

Would it make sense to optionally render this component here? This way, when we implement the component's content, we avoid assuming a potential null tier and simplify the rendering logic. wdyt?